summary_workflows/quantification/quantification_dockers/q_consolidation/manage_assessment_data.py

```diff
@@ -80,7 +166,7 @@ def generate_manifest(data_dir,output_dir, participant_data):
                 metric_Y = "precision"
```

Comment:
This part (line 163-166) has to be updated, but I'm not sure how. We have only one metric "correlation". What will be the other axis (if this means we need a different type of plot, we'd have to change that too. But I don't know where)